react/features/whiteboard/reducer.ts

```diff
@@ -0,0 +1,84 @@
+// @ts-ignore
+import { ReducerRegistry } from '../base/redux';
+
+// @ts-ignore
+import { DISABLE_WHITEBOARD, ENABLE_WHITEBOARD, SET_USERNAME_STATUS } from './actionTypes';
+
+export interface IWhiteboardState {
+
+    /**
+     * The whiteboard collaboration details.
+     */
+    collabLink: { roomId: string; roomKey: string; } | null;
```

Comment:
Bad name, it's not a link.

Reply:
Right, how about collabDetails? I need to pass this object to excalidraw.

Reply:
👍